pkg/proxy/ipvs/proxier.go

```diff
@@ -666,6 +670,26 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 		}
 	}
 
+	var nfErrCount int
+	for _, nfctmodule := range nfConntrackModules {
```

Comment:
The same here: https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/ipvs/proxier.go#L656-L669 ?